src/contracts/core/DelegationManagerStorage.sol

```diff
@@ -99,13 +95,6 @@ abstract contract DelegationManagerStorage is IDelegationManager {
     /// See conversation here: https://github.com/Layr-Labs/eigenlayer-contracts/pull/365/files#r1417525270
     address private __deprecated_stakeRegistry;
 
-    /// @notice Mapping: AVS => operator => enum of operator status to the AVS
-    mapping(address => mapping(address => OperatorAVSRegistrationStatus)) public avsOperatorStatus;
-
-    /// @notice Mapping: operator => 32-byte salt => whether or not the salt has already been used by the operator.
-    /// @dev Salt is used in the `registerOperatorToAVS` function.
-    mapping(address => mapping(bytes32 => bool)) public operatorSaltIsSpent;
```

Comment:
Do we need to deprecate these variables instead? if they're internal and unused, i don't think it would contribute to bytecode size, though maybe i should double check that.

Reply:
i'm wondering what happens if we upgrade the current m2 deployment to something that doesn't have these variables when they've already been used -- if we have an m3 down the line that reintroduces variables at these slots, we might find storage collisions?

Reply:
OTOH maybe it doesnt matter since we'll deprecate Goerli well before m3

Reply:
These storage variables have not been set in the DM yet on mainnet so should be safe to remove. For second goerli upgrade however I need to doublecheck...

Reply:
These variables weren't in the initial Goerli deployment either, at least if I recall correctly.
so that would mean super safe to delete.